configure.yml

```diff
@@ -113,7 +113,22 @@
           tags:
             - workflows
 
-        - name: destroy awx job templates
+        - name: changes detection in jobs templates
+          uri:
+            url: "{{ awx_local_url }}/api/v2/job_templates/"
+            method: GET
+            body_format: json
+            user: "{{ awx_admin_user | default('admin') }}"
+            password: "{{ awx_admin_pass }}"
+            force_basic_auth: yes
+            status_code: 204
+          register: list_templates_query
+
+        - name: set template list type dict
+          set_fact:
+            list_templates: "{{ list_templates_query.json.results | list | list_to_dict('kind') }}"
```

Comment:
Change to:
list_templates: "{{ list_templates_query.json.results | list }}"

Reply:
No puedo seguir esta recomendacion, dado que hice las tareas con la variable definida list_templates. Siempre quieres el cambio?

Reply:
si, todos los demas cambios hacen referencia a esta definicion